SeleniumPracticalExercise/PageObjects/EmployeeInformationPage.cs

```diff
@@ -26,8 +26,7 @@ public void ClickAdd()
         }
 
         /// <summary>
-        /// search for employee ID field and insert stored ID
-        /// then we click search button
+        /// search for employee ID textbox field and insert ID
```

Comment:
This can be simply, "Sets the employee ID to the provided value" or something like that... See FindElement() or SendKeys() in BasePageLocal.cs.

Reply:
"Sets the employee Id to employeeId" is too literal. We shouldn't need to restate exactly what the code does because the code is there for them to see if they need that level of detail.

It should be, "Sets the employee ID to the provided value" or "Sets the employee ID."